packages/core/src/rules/common/operation-tag-defined.ts

```diff
@@ -11,6 +11,13 @@ export const OperationTagDefined: Oas3Rule | Oas2Rule = () => {
       definedTags = new Set((root.tags ?? []).map((t) => t.name));
     },
     Operation(operation: Oas2Operation | Oas3Operation, { report, location }: UserContext) {
+      if (!operation?.hasOwnProperty('tags')) {
+        report({
+          message: `Operation tags should be defined`,
+          location: location.child('tags').key(),
```

Comment:
If there's no `operation.tags`, then the location couldn't be `tags`, I believe. Would the following work?

```suggestion
          location: location.key(),
```

Reply:
let's try it. .the snapshot seemed to work ok, but i'll try updating it. your comment makes sense